### DISCUSSION TITLE
Can't save edited pages or update wiki version

### DISCUSSION THREAD

**Discussion by charm-games** [requarks/wiki#4501] [Error / Bug Report]
We are on version 2.5.170 and all of a sudden we started getting the issue where we can't save any pages. Any time we edit them and they try to refresh we just get stuck in the rendering wait loop. 

We tried updating to 2.5.219 but then we get an error:
![wiki-js-update-error](https://user-images.githubusercontent.com/17397197/134950759-e14e9f8f-f4c3-4a32-b67b-af57d0952245.png)

Any suggestions?

**Comment by Phil-hostwiki** [comment]
Did you install the `wiki-update-companion`? Is it running?
